tests/app/main/views/test_services.py

```diff
@@ -31,8 +31,20 @@ def test_service_find_returns_404_for_missing_service_id(self):
         assert response.status_code == 404
 
 
+@mock.patch('app.main.views.services.data_api_client', autospec=True)
 class TestServiceView(LoggedInApplicationTest):
-    @mock.patch('app.main.views.services.data_api_client')
+
+    find_audit_events_api_response = {'auditEvents': [
+        {
```

Comment:
Could flesh these out a bit beyond minimal data to make it clear what we expect real responses to look like.  Doesn't need to have absolutely all fields, but something like this might be nice to give a bit more context:
```
{
      "createdAt": "2017-11-17T16:30:30.837267Z",
      "data": {
          "new_status": "disabled",
          "old_status": "published",
          "serviceId": "314159265",
      },
      "type": "update_service_status",
      "user": "anne.admin@example.com" 
}
```

Reply:
Sure - I'll include the AuditType as well so it's clear we're returning a `update_service_status` event rather than a general edit.